styles.css

```diff
@@ -0,0 +1,331 @@
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+/* Texts headings and Paragraphs Font Poppins Fonts Styling */
+.popheading {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.popp13 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.popp15 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.abouth {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.aboutp {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+}
+
+.abouts {
+  text-decoration: none;
+  color: #396df2;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border: 2px solid lightgray;
+  border-radius: 5px;
+  display: inline-block;
+  padding: 18px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border: 1px solid gray;
+  border-radius: 8px;
+  margin-top: 40px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 90%;
+
+  /* height: 600px; */
+  background-color: white;
+}
+
+.imgdiv {
+  flex: 100%;
+  margin-left: 20px;
+  margin-top: 20px;
+}
+
+.tonic {
+  flex: 100%;
+  margin-left: 20px;
+  margin-bottom: 20px;
+}
+
+.htcj {
+  background: #ebebff;
+  border-radius: 8px;
+  display: inline-block;
+  padding: 6px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+}
+
+.btn,
+.btn3,
+.btn2,
+.btn4 {
+  border: 2px solid lightgray;
+  border-radius: 5px;
+  display: inline-block;
+  padding: 18px;
+}
+
+.btn a,
+.btn3 a,
+.btn2 a,
+.btn4 a {
+  text-decoration: none;
+  color: #396df2;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.btn:hover,
+.btn3:hover,
+.btn2:hover,
+.btn4:hover {
+  background: #6070ff;
+  color: white;
+}
+
+.btn a:hover,
+.btn2 a:hover,
+.btn3 a:hover,
+.btn4 a:hover {
+  background: #6070ff;
+  color: #fff;
+}
+
+.btn:active,
+.btn3:active,
+.btn2:active,
+.btn4:active {
+  background: #2230d2;
+  color: white;
+}
+
+.btn a:active,
+.btn2 a:active,
+.btn3 a:active,
+.btn4 a:active {
+  background: #2230d2;
+  color: #fff;
+}
+
+.container2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border: 1px solid gray;
+  border-radius: 8px;
+  margin-top: 30px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 90%;
+
+  /* height: 600px; */
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your project. It is unprofessional and makes your code unclean. :+1:

Reply:
Commented piece of code removed